components/style/properties/data.py

```diff
@@ -220,12 +230,19 @@ def new_style_struct(self, *args, **kwargs):
     def active_style_structs(self):
         return [s for s in self.style_structs if s.additional_methods or s.longhands]
 
+    def add_prefixed_aliases(self, property_data):
+        # FIXME Servo's DOM architecture doesn't support vendor-prefixed properties.
```

Comment:
Also, for future reference we do not want moz or unspecced webkit prefixes in Servo unless absolutely necessary. Only prefixes from the compat spec (and change the spec if a property seems necessary). We may want to split this attribute into two or use ifs at the usage site when we start thinking about servo.